apps/gallery/test/marionette/lib/gallery.js

```diff
@@ -21,7 +21,7 @@ Gallery.ORIGIN = 'app://gallery.gaiamobile.org';
 Gallery.Selector = Object.freeze({
   thumbnail: '.thumbnail',
   fullscreenView: '#fullscreen-view',
-  thumbnailsView: '#thumbnail-list-view',
+  thumbnailsView: '#thumbnail-footers > .thumbnails-list',
```

Comment:
I take it back, after looking at the test thumbnailsView name is confusing and should be updated to thumbnailListFooter